### PR TITLE
Feature/#068 paginacion productos

### PR DESCRIPTION
**Pull request by juradosalado**

### CODE REVIEW

File: static/styles/home.css [hunk 1]
```diff
@@ -67,4 +67,22 @@ img {
...
+.navigation li a{
+  color: white;
+  background-color: var(--bs-indigo);
+}
+
+.pagination{
+  margin: auto;
+}
+.navigation li a:hover{
+  color: black;
```
Comment (migromarj) [inline static/styles/home.css:86]: I would change the color of the text when hovering to a slightly lighter color, #ffc107 for example

**Review by migromarj — APPROVED** [review]
The page works correctly and the pagination is spectacular 😮 💯 , I have made a suggestion about the color of the pagination text when hovering.

**Review by IsmaelP19 — APPROVED** [review]
Everything looks great!! 🎉🙌
I agree with @migromarj, you should change the color of the text when hovering to a lighter color to make it easier to read.

File: DTechnology/views.py [hunk 2]
```diff
@@ -100,8 +101,17 @@ def home(request):
...
+    if request.GET.get('search'):
+        search = request.GET.get('search')
+    if request.GET.get('category'):
+        active_category = request.GET.get('category')
+    if request.GET.get('department'):
+        active_department = request.GET.get('department')
+    if request.GET.get('producer'):
+        active_producer = request.GET.get('producer')
+    [products, page_obj] = get_products(active_category, active_department, active_producer, search, request, page_size=12)
+    print(products)
```
Comment (IsmaelP19) [inline DTechnology/views.py:114]: You should delete this line.

**Review by IsmaelP19 — APPROVED** [review]
All tests running OK. The hovering effect at the pagination buttons is great now. Well done! 🥳🎉

**Review by migromarj — APPROVED** [review]
Thanks for considering the suggestions, the pagination is perfect and everything works correctly. Great job Juan 💪🏻 🥇

**Pull request merged**